packages/paste-website/src/utils/SimpleStorage.ts

```diff
@@ -0,0 +1,35 @@
+const BASE_KEY = 'paste-website';
+
+let storage: Storage = {
+  getItem: () => null,
+  setItem: () => null,
+  removeItem: () => null,
+  clear: () => null,
+  length: 0,
+  key: () => null,
+};
+
+try {
+  storage = window.localStorage;
+  // eslint-disable-next-line unicorn/prefer-optional-catch-binding
+} catch (error) {
+  // Ignore
+}
+
+const namespaceKey = (key: string): string => `${BASE_KEY}/${key}`;
+
+// Simple localStorage wrapper w/ namespace support
```

Comment:
Should we start trying to encourage JSDoc style comments? Just as a nice DX it'll give you nice autocomplete information in VS Code.